deblur/workflow.py

```diff
@@ -438,16 +485,22 @@ def launch_workflow(seqs_fp, working_dir, read_error, mean_error, error_dist,
         filepath(s) to SortMeRNA indexed database for artifact removal
     negate: boolean
         discard all sequences aligning to the ref_fp database
-    threads: integer, optional
-        number of threads to use for SortMeRNA
+    threads_per_sample: integer, optional
```

Comment:
I don't think you need to change the parameter name here - if launch workflow is already running in a per sample basis, is already understood that this 'threads' parameter only applies in a per sample basis.